libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/summary/SummaryBottomSheet.java

```diff
@@ -163,6 +167,15 @@ public void setTimeFormat(@NavigationTimeFormat.Type int type) {
     this.timeFormatType = type;
   }
 
+  /**
+   * Sets the rounding increment which will be used for rounding small distances
```

Comment:
Let's be more specific with the javadoc here - maybe provide an example to illustrate.